modules.json

```diff
@@ -367,4 +367,28 @@
             "ucuser": "Jimbo Wales"
     	}
     }
+    {
```

Comment:
The code sample you are writing for is for a module that supports POST requests. We add parameters to modules.json for only those modules that support GET requests.